.github/workflows/pythonapp.yml

```diff
@@ -8,6 +8,8 @@ on:
     branches:
       - master
   pull_request:
+    branches:
```

Comment:
I don't think we need to add these two lines. I think we want checks on all pull requests.